doc/tutorial.md

```diff
@@ -77,7 +84,7 @@ The successful result will look like:
 Estimated gas: Estimate (gas_used: 69, vm_status_code: EXECUTED)
 ```
 
-## Publishing Modules
+### Publication 
```

Comment:
While I agree this is a more fancy word, I would stay with `Publishing modules`, since even the `aptos` command uses such wording:
```
  publish
          Publishes the modules in a Move package to the Aptos blockchain
```